src/redux/migrations.ts

```diff
@@ -1877,4 +1877,28 @@ export const migrations = {
       'mtwAddress'
     ),
   }),
+  229: (state: any) => {
+    const transactionsByNetworkId: any = {}
+    for (const networkId of Object.keys(state.transactions.transactionsByNetworkId)) {
+      const transactions = []
+      for (const tx of state.transactions.transactionsByNetworkId[networkId]) {
+        ;(networkId === NetworkId['arbitrum-one'] || networkId === NetworkId['arbitrum-sepolia']) &&
```

Comment:
nit: code golf
`[NetworkId['arbitrum-one'], NetworkId['arbitrum-sepolia']].includes(networkId) &&`

Reply:
@MuckT this gives me a type error since `const networkId of Object.keys(state.transactions.transactionsByNetworkId)` is being typed as a string